src/content/mod_configuration.cpp

```diff
@@ -21,16 +21,26 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "log.h"
 #include "settings.h"
 #include "filesys.h"
+#include "gettext.h"
 
-void ModConfiguration::printUnsatisfiedModsError() const
+
+std::string ModConfiguration::getUnsatisfiedModsError() const
 {
+	std::ostringstream error;
+	error << gettext("Some mods have unsatisfied dependencies:") << std::endl;
+
 	for (const ModSpec &mod : m_unsatisfied_mods) {
-		errorstream << "mod \"" << mod.name
-					<< "\" has unsatisfied dependencies: ";
+		error << " - " << fmtgettext("%s is missing:", mod.name.c_str());
```

Comment:
needs an annotation or nobody will ever translate this correctly

Reply:
How do I do annotations?

Reply:
`//~ text here`

Reply:
@rubenwardy :eyes: